plugins/services/src/js/containers/volume-detail/VolumeDetail.js

```diff
@@ -54,6 +54,10 @@ class VolumeDetail extends React.Component {
       />
     );
 
+    const sizeLabel = volume.type === 'External'
```

Comment:
I agree with @nLight - a ternary in this case is the easiest to read and cleanest IMO.